### DISCUSSION TITLE
希望增加对文言文的支持

### DISCUSSION THREAD

**Discussion by qingtom** [immersive-translate/immersive-translate#373] [Ideas]
### 新功能 | New Feature

1. 增加对文言文的支持
2. 白话文到文言文

### 为什么您想要添加这些功能 | Why you want to add these features

1. 古文学习过程中如果有自动的翻译，对学习效率有很大的帮助

### 提议 | Proposal

微软提供文言文翻译的接口，但不知道是否收费。

**Comment by versun** [comment]
微软还有这接口呀，🐮

**Comment by qingtom** [comment]
https://www.bing.com/translator
<img width="698" alt="image" src="https://user-images.githubusercontent.com/209574/233227853-23176709-de8d-442e-a485-d99942e8d728.png">

**Reply by versun** [reply]
其实插件上有对文言文的支持，但貌似在源语言的识别上有点问题，会尝试修复的。
![image](https://user-images.githubusercontent.com/2398708/233242556-17e6a517-6b8e-4424-bd91-fe7b4952bfda.png)

**Reply by qingtom** [reply]
果然有，是我没看仔细😅
